src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -209,54 +207,45 @@ export default function ActionsPanel(): JSX.Element {
 
   return (
     <Panel
+      drawer={
+        <Button
+          appearance="positive"
+          disabled={disableSubmit}
+          onClick={handleSubmit}
+        >
+          Run action
+        </Button>
+      }
       width="narrow"
-      panelClassName="actions-panel"
       data-testid={TestId.PANEL}
       title={generateTitle()}
       onRemovePanelQueryParams={handleRemovePanelQueryParams}
     >
-      <>
```

Comment:
This is mostly indentation changes except where I've moved the button to the draw prop.